tests/test_fixtures/rule_based_profiler/example_notebooks/DataAssistants_Instantiation_And_Running-OnboardingAssistant-Pandas.ipynb

```diff
@@ -0,0 +1,911 @@
+   "outputs": [],
+   "source": [
+    "checkpoint_config: dict = {\n",
+    "    \"name\": \"my_checkpoint\",\n",
+    "    \"config_version\": 1,\n",
+    "    \"class_name\": \"SimpleCheckpoint\",\n",
+    "    \"validations\": [\n",
+    "        {\n",
+    "            \"batch_request\": single_batch_batch_request,\n",
+    "            \"expectation_suite_name\": \"taxi_data_2019_suite\",\n",
+    "        }\n",
+    "    ],\n",
+    "     \"action_list\":[\n",
+    "            {\n",
+    "                \"name\": \"update_data_docs\",\n",
+    "                \"action\": {\n",
+    "                    \"class_name\": \"UpdateDataDocsAction\",\n",
+    "                },\n",
+    "            },\n",
+    "        ],\n",
+    "}\n",
+    "data_context.add_checkpoint(**checkpoint_config)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3a718ee2-1971-495e-8113-8704752fd3c8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_context.add_checkpoint(**checkpoint_config)\n",
+    "results = data_context.run_checkpoint(\n",
+    "    checkpoint_name=\"my_checkpoint\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "10a18e22",
+   "metadata": {},
+   "source": [
+    "### Examine results by looking at DataDocs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b353ea25",
+   "metadata": {},
+   "source": [
+    "As you can see the results of the `Checkpoint` were not successful, which means the January 2020 data did not fall within the ranges predicted by the 2019 data. To examine this more closely, you can [open Data Docs here](great_expectations/uncommitted/data_docs/local_site/index.html). If the link doesn't work, from the location where this Notebook is located, you can go to `great_expectations/uncommitted/data_docs/local_site/index.html`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bfc2cc49-cd02-44db-9079-d0baed7ffe73",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results.success"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0d02f5e8",
+   "metadata": {},
+   "source": [
+    "## Optional: Clean-up Directory\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f973a7ef",
+   "metadata": {},
+   "source": [
+    "As part of running this notebook, the `DataAssistant` will create a number of ExpectationSuite configurations in the `great_expectations/expectations/tmp` directory. Optionally run the following cell to clean up the directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4857e285",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import shutil, os\n",
+    "# shutil.rmtree(f\"great_expectations/expectations/\")\n",
+    "# shutil.rmtree(f\"great_expectations/checkpoints/\")\n",
+    "# shutil.rmtree(f\"great_expectations/uncommitted/\")\n",
+    "# os.remove(\"great_expectations/expectations/.ge_store_backend_id\")\n"
```

Comment:
We should leave the /expectations/ folder and the .ge_store_backend_id to make sure that we don't inadvertently create a new data_context_id when this user continues working with the library. Also why are we deleting checkpoints and uncommitted?

Reply:
Great point. I think the original intent was to "clean up" the files that were generated when the notebook is run, but this was a bit like taking a sledge hammer to the `great_expectations/` folder.  I'm going to see if I can make this more precise :)

Reply:
The section now only removes the Suite and Checkpoint that was created
``` 
import os
os.remove('great_expectations/checkpoints/my_checkpoint.yml')
os.remove('great_expectations/expectations/taxi_data_2019_suite.json')
```